lib/Doctrine/Common/Annotations/NamedArgumentConstructorAnnotation.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Doctrine\Common\Annotations;
+
+/**
+ * Marker interface for PHP7/PHP8 compatible support
+ * for named arguments (and constructor property promotion).
```

Comment:
Should constructor property promotion really be mentioned here? You did not write a test for it, so I believe this means it does not really come into play here: it will work the same regardless of whether you use that feature or not.